twtr/src/integration-test/groovy/twtr/MessageFunctionalSpec.groovy

```diff
@@ -0,0 +1,386 @@
+        def response1 = restClient.get(path: "/accounts/${goodId1}/messages/1")
+        def response2 = restClient.get(path: "/accounts/${goodId1}/messages/2")
+
+        then:
+        response1.status == 200
+        response1.data
+        response1.data.handle == response.data.handle[1]
+        response1.data.messageText == response.data.messageText[1]
+        response1.data.dateCreated == response.data.dateCreated[1]
+        response2.status == 200
+        response2.data
+        response2.data.handle == response.data.handle[0]
+        response2.data.messageText == response.data.messageText[0]
+        response2.data.dateCreated == response.data.dateCreated[0]
+    }
+
+    def 'create a tweet with #message3 to account with #goodId1'() {
+        when:
+        def response = restClient.post(path: '/message/tweet', query: [accountId: goodId1],
+                contentType: 'application/json', body: [messageText: message3])
+        goodDateCreated[2] = response.data.dateCreated
+
+        then:
+        response.status == 201
+        response.data
+
+        when:
+        response = restClient.get(path: "/accounts/${goodId1}")
+
+        then:
+        response.status == 200
+        response.data
+        response.data.id == goodId1
+        response.data.messageCount == 3
+
+        when:
+        response = restClient.get(path: "/accounts/${goodId1}/messages")
+
+        then:
+        response.status == 200
+        response.data
+
+        when:
+        def response1 = restClient.get(path: "/accounts/${goodId1}/messages/1")
+        def response2 = restClient.get(path: "/accounts/${goodId1}/messages/2")
+        def response3 = restClient.get(path: "/accounts/${goodId1}/messages/3")
+
+        then:
+        response1.status == 200
+        response1.data
+        response1.data.handle == response.data.handle[2]
+        response1.data.messageText == response.data.messageText[2]
+        response1.data.dateCreated == response.data.dateCreated[2]
+        response2.status == 200
+        response2.data
+        response2.data.handle == response.data.handle[1]
+        response2.data.messageText == response.data.messageText[1]
+        response2.data.dateCreated == response.data.dateCreated[1]
+        response3.status == 200
+        response3.data
+        response3.data.handle == response.data.handle[0]
+        response3.data.messageText == response.data.messageText[0]
+        response3.data.dateCreated == response.data.dateCreated[0]
+    }
+
+
+    def 'return the most recent messages from an account with #goodId1'() {
+        given: 'create 12 more messages for #goodId1 to make 15 messages total'
+        def response
+        for (def i = 4; i <= 15; i++) {
+            response = restClient.post(path: '/message/tweet', query: [accountId: goodId1],
+                    contentType: 'application/json', body: [messageText: messageTemplate + i])
+            goodDateCreated[i - 1] = response.data.dateCreated
+        }
```

Comment:
you can always do another commit and it will just add it to the PR that's open automatically.  But, I'm gonna merge this one.